api/Clients.json

```diff
@@ -0,0 +1,357 @@
+              "version_added": "45"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "38"
+            },
+            "opera_android": {
+              "version_added": "38"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "matchAll": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Clients/matchAll",
+          "support": {
+            "webview_android": {
+              "version_added": "47",
+              "notes": "<code>Client</code> objects returned in most recent focus order."
+            },
+            "chrome": {
+              "version_added": "47",
+              "notes": "<code>Client</code> objects returned in most recent focus order."
+            },
+            "chrome_android": {
+              "version_added": "47",
+              "notes": "<code>Client</code> objects returned in most recent focus order."
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "44",
+                "notes": "Service workers (and Push) have been disabled in the Firefox 45 & 52 Extended Support Releases (ESR)."
+              },
+              {
+                "version_added": "54",
+                "notes": "<code>Client</code> objects returned in most recent focus order."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "44"
+              },
+              {
+                "version_added": "54",
+                "notes": "<code>Client</code> objects returned in most recent focus order."
+              }
+            ],
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "32"
+            },
+            "opera_android": {
+              "version_added": "32"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": "4.0"
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "includeUncontrolled_option": {
```

Comment:
Thanks for adding this, however it needs to be nested underneath "matchAll".

- matchAll
  - __compat
  - includeUncontrolled_option